### PR TITLE
Xeve vui support

### PR DESCRIPTION
**Pull request by p-chirag**

### CODE REVIEW

File: app/xeve_app.c [hunk 1]
```diff
@@ -633,6 +633,261 @@ static int update_rc_param(ARGS_PARSER * args, XEVE_PARAM * param)
     return 0;
 }
 
+
+static int vui_param_check(XEVE_PARAM * param)
```
Comment (kpchoi) [inline app/xeve_app.c:637]: Where do you call this function?

Comment (p-chirag) [reply app/xeve_app.c:637]: Added the call, I think it was deleted out while revert and again pushing it back. pls check now

**Comment by kpchoi** [comment]
I found the following error in this code. it needs to be solved.

```
/home/ulsiguya/src/xeve/code/app/xeve_app.c: In function 'y4m_is_regular_file':
/home/ulsiguya/src/xeve/code/app/xeve_app.c:78:20: error: implicit declaration of function '_fileno'; did you mean 'fileno'? [-Werror=implicit-function-declaration]
   78 |     if( y4m_fstat( _fileno( filehandle ), &file_stat ) )
      |                    ^~~~~~~
      |                    fileno
cc1: all warnings being treated as errors
make[2]: *** [app/CMakeFiles/xeve_app.dir/build.make:63: app/CMakeFiles/xeve_app.dir/xeve_app.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:199: app/CMakeFiles/xeve_app.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

**Comment by washish** [comment]
> @mss-park @p-chirag I found the following error in this code when I compile code under Linux. it needs to be solved.
> 
> ```
> /home/ulsiguya/src/xeve/code/app/xeve_app.c: In function 'y4m_is_regular_file':
> /home/ulsiguya/src/xeve/code/app/xeve_app.c:78:20: error: implicit declaration of function '_fileno'; did you mean 'fileno'? [-Werror=implicit-function-declaration]
>    78 |     if( y4m_fstat( _fileno( filehandle ), &file_stat ) )
>       |                    ^~~~~~~
>       |                    fileno
> cc1: all warnings being treated as errors
> make[2]: *** [app/CMakeFiles/xeve_app.dir/build.make:63: app/CMakeFiles/xeve_app.dir/xeve_app.c.o] Error 1
> make[1]: *** [CMakeFiles/Makefile2:199: app/CMakeFiles/xeve_app.dir/all] Error 2
> make: *** [Makefile:130: all] Error 2
> ```
Fixed. Please check

**Comment by kpchoi** [comment]
@mamthan 
Hi, if you think this PR is too much outdated, I could close this PR.
can we close?

**Comment by mamthan** [comment]
@kpchoi Yes, Master Choi. Please close this PR.

**Pull request closed without merging**